### PR TITLE
Backup cron

### PR DESCRIPTION
**Pull request by antoine-de**
port https://github.com/etalab/transport-history/ to elixir

add a daily cron job that backup every GTFS/Netex (for the moment) to S3.

### CODE REVIEW

File: apps/transport/lib/transport/history.ex [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+  require Logger
+
+  def backup_resources() do
+    if Application.get_env(:ex_aws, :access_key_id) == nil
+      || Application.get_env(:ex_aws, :secret_access_key) == nil do
+      # if the cellar credential are missing, we skip the whole history backup
+        Logger.warn("no cellar credential set, we skip to resource backup")
+    else
+      resources_to_backup = Resource
+      |> where([r], not is_nil(r.url) and not is_nil(r.title) and (ilike(r.format, "GTFS") or ilike(r.format, "netex")))
```
Comment (Tristramg) [inline apps/transport/lib/transport/history.ex]: Check by equality, not like (btw, without `%` it’s an equality)

Comment (antoine-de) [reply apps/transport/lib/transport/history.ex]: but that's a *i*like here, so that's different no ?

but yes I think we can strictly compare

**Comment by Tristramg** [comment]
yes, but caps are ok and are sanitized (we should use constants btw, but
that’s a more general problem)

**Comment by antoine-de** [comment]
now the history is presented as:
image


I think we still need to add a marker on the current resources, but maybe that can wait another PR ?

**Pull request merged**